metrics.go

```
@@ -192,7 +192,7 @@ func reportTiming(ctx context.Context, fn string, tags ...Tags) (context.Context
 
 		clientMux.RLock()
 		defer clientMux.RUnlock()
-		client.Timing("func.timing", d, stopTagArray, 1)
+		client.Histogram("func.timing", float64(d.Milliseconds()), stopTagArray, 1)
```

golangci-lint (2.12.1)

[error] Error return value of `client.Histogram` is not checked

(errcheck)